examples/permissioned-voting-application/scripts/voting.js

```diff
@@ -0,0 +1,77 @@
+const { executeTransaction, TransactionType, SignType } = require("algob");
+
+/**
+ * Description: Converts Integer into Bytes Array
+ */
+function getInt64Bytes(x) {
+  let y= Math.floor(x/2**32);
+  const byt = [y,(y<<8),(y<<16),(y<<24), x,(x<<8),(x<<16),(x<<24)].map(z=> z>>>24)
+	return new Uint8Array(byt);
+}
+
+async function run(runtimeEnv, deployer) {
+
+  const masterAccount = deployer.accountsByName.get("master-account")
+  const aliceAccount = deployer.accountsByName.get("alice-account");
+  const votingAdminAccount = deployer.accountsByName.get("voting-admin-account");
+
+  await executeTransaction(deployer, {type:TransactionType.TransferAlgo, sign:SignType.SecretKey, 
+  fromAccount: masterAccount, toAccountAddr:votingAdminAccount.addr, amountMicroAlgos: 200000000, payFlags: {}});
+
+	// Create ASA - Vote Token
+	const asaInfo = await deployer.deployASA("vote-token", {
+	creator: votingAdminAccount});
+	console.log(asaInfo);
```

Comment:
please fix the indentation

Reply:
I don't why this happens, if you see this file using `view file`, it will show you correct indentation but review changes shows broken indentation.